src/main/java/com/codejoust/main/service/generators/PythonDefaultCodeGeneratorService.java

```diff
@@ -27,8 +27,12 @@ public String getDefaultCode(List<ProblemInput> problemInputs, ProblemIOType out
         }
         methodLineBuilder.append("):");
 
+        // Add a method header comment to describe the return type.
+        String returnComment = String.format("\t# The solve method should return the type %s", outputType.getClassType().getSimpleName());
+
```

Comment:
Instead a comment actually perhaps we could use Python's built-in [type hinting](https://docs.python.org/3/library/typing.html)? Didn't realize this was a thing till now but also saw that this is what LeetCode does if you select Python 3 as the language

Could also be a good use of `typeInstantiationToString` since I don't think it's being used currently

Also as a side note I realized we're defining the Python method wrong lol, I think we need a `self` as the first parameter otherwise it's a static method, so sometime in the future we could probably update that (since we'll need to fix tester too to call it as an instance method rather than currently as a static method)

Reply:
> Instead a comment actually perhaps we could use Python's built-in type hinting? Didn't realize this was a thing till now but also saw that this is what LeetCode does if you select Python 3 as the language
>
> Could also be a good use of typeInstantiationToString since I don't think it's being used currently
That's a great idea, I'll add that now!

> Also as a side note I realized we're defining the Python method wrong lol, I think we need a self as the first parameter otherwise it's a static method, so sometime in the future we could probably update that (since we'll need to fix tester too to call it as an instance method rather than currently as a static method)
I think we could fix that here as well @alankbi - maybe I could add that to [tester #22](https://github.com/CodeJoustHQ/tester/pull/22) since that PR is very small and could use another addition?

Reply:
Done 👍